<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendReceiveManager</class>
 <widget class="QMainWindow" name="SendReceiveManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="clientVerticalBoxLayout1">
      <item>
       <widget class="QLabel" name="clientLabel1">
        <property name="text">
         <string>Client [1]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="clientListWidget1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="clientLineEdit1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="clientVerticalBoxLayout2">
      <item>
       <widget class="QLabel" name="clientLabel2">
        <property name="text">
         <string>Client [2]</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="clientListWidget2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="clientLineEdit2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="serverVerticalBoxLayout">
      <item>
       <widget class="QLabel" name="serverLabel">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="serverListWidget"/>
      </item>
      <item>
       <widget class="QLineEdit" name="serverLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
